tools/const_finder/bin/main.dart

```diff
@@ -58,7 +58,14 @@ void main(List<String> args) {
         help: 'Print usage and exit');
 
   final ArgResults argResults = parser.parse(args);
-  T getArg<T>(String name) => argResults[name] as T;
+  T getArg<T>(String name) {
+    try {
+      return argResults[name] as T;
+    } catch (err) {
```

Comment:
Can you catch a specific exception here?

Reply:
It's a private `_CastError`. Also, it's being rethrown, this is just a usability convenience to let the user know which flag they didn't provide.

Reply:
I can update this STDERR to be even more user-friendly...

Reply:
Actually nvm, let me just mark the options mandatory.